frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1162,7 +1162,8 @@ function ContextMenuInner(propsWithInputRef: Props) {
         contextMenuPosition == null ||
         volumeTracing == null ||
         !hasNoFallbackLayer ||
-        !volumeTracing.hasSegmentIndex
+        !volumeTracing.hasSegmentIndex ||
+        props.additionalCoordinates != null // TODO change once statistics are available for nd-datasets
```

Comment:
maybe link to this todo-comment already inside the other PR (and add a checkbox to the GH comment) so that we don't forget about it.